lighthouse-core/gather/gatherers/dobetterweb/optimized-images.js

```diff
@@ -169,36 +173,42 @@ class OptimizedImages extends Gatherer {
    * @param {Array<SimplifiedNetworkRecord>} imageRecords
    * @return {Promise<LH.Artifacts['OptimizedImages']>}
    */
-  computeOptimizedImages(driver, imageRecords) {
+  async computeOptimizedImages(driver, imageRecords) {
     /** @type {LH.Artifacts['OptimizedImages']} */
-    const result = [];
-
-    return imageRecords.reduce((promise, record) => {
-      return promise.then(results => {
-        return this.calculateImageStats(driver, record)
-          .catch(err => {
-            // Track this with Sentry since these errors aren't surfaced anywhere else, but we don't
-            // want to tank the entire run due to a single image.
-            // @ts-ignore TODO(bckenny): Sentry type checking
-            Sentry.captureException(err, {
-              tags: {gatherer: 'OptimizedImages'},
-              extra: {imageUrl: URL.elideDataURI(record.url)},
-              level: 'warning',
-            });
-            return {failed: true, err};
-          })
-          .then(stats => {
-            if (!stats) {
-              return results;
-            }
-
-            return results.concat(Object.assign(stats, record));
-          });
-      });
-    }, Promise.resolve(result));
-  }
+    const results = [];
+
+    for (const record of imageRecords) {
+      try {
+        const stats = await this.calculateImageStats(driver, record);
```

Comment:
throwing this out there... maybe we should let these race and promise.all them instead. would probably be faster.

but i cant remember if I've seen this gatherer being particularly slow..

Reply:
k sg. then yah nvm. we'll keep it running serially.